k8s.gcr.io/k8s-staging-cluster-api/manifest.yaml

```diff
@@ -0,0 +1,9 @@
+# google group for gcr.io/k8s-staging-cluster-api is k8s-infra-gcr-staging-cluster-api@googlegroups.com
+registries:
+  src: gcr.io/k8s-staging-cluster-api
+  dest: gcr.io/k8s-staging
+service-account: service-account@kubernetes-public.iam.gserviceaccount.com
+images:
```

Comment:
You should do `images: []` to denote an empty list.

Reply:
ack